src/App.js

```diff
@@ -1,29 +1,20 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
+import {
+  BrowserRouter as Router,
+  Route,
+} from 'react-router-dom';
+import Intro from './components/Intro.jsx';
```

Comment:
Can we add a new line between the react and react router imports and our personal imports?